framework/src/modules/chain/chain.js

```diff
@@ -110,25 +114,25 @@ module.exports = class Chain {
 				this.options.genesisBlock.communityIdentifier,
 			);
 
-			const self = this;
-			this.scope = {
-				config: self.options,
-				genesisBlock: { block: self.options.genesisBlock },
-				registeredTransactions: self.options.registeredTransactions,
-				sequence: new Sequence({
-					onWarning(current) {
-						self.logger.warn('Main queue', current);
-					},
-				}),
-				components: {
-					storage: this.storage,
-					logger: this.logger,
-				},
-				channel: this.channel,
-				applicationState: this.applicationState,
+			this.config = this.options;
+			this.genesisBlock = { block: this.config.genesisBlock };
+			this.registeredTransactions = this.options.registeredTransactions;
+
+			this.components = {
+				storage: this.storage,
+				logger: this.logger,
 			};
 
-			await bootstrapStorage(this.scope, global.constants.ACTIVE_DELEGATES);
+			this.sequence = new Sequence({
+				onWarning(current) {
+					this.components.logger.warn('Main queue', current);
+				},
+			});
+
+			await bootstrapStorage(
+				{ components: this.components },
+				global.constants.ACTIVE_DELEGATES,
```

Comment:
this can be changed to reference value from `this.config`, and `global` should be removed

Reply:
Done!